contracts/nns/contract.go

```diff
@@ -60,7 +60,7 @@ const (
 	maxTXTRecordLength = 255
 	// maxRecordID is the maximum value of record ID (the upper bound for the number
 	// of records with the same type).
-	maxRecordID = 255
+	maxRecordID = 16
```

Comment:
do we have some ref why this number is good? is it your idea? is it from some original version of NNS?

Reply:
No. Yes. No. We definitely need to fix ID overflow problem, but using a lot of records can be a problem performance-wise (likely signifying some design issues). So start small, extend if needed.